.github/workflows/release.yml

```
@@ -0,0 +1,110 @@
+      uses: softprops/action-gh-release@v1
+      with:
+        files: |
+          target/px-submission-tool-*.zip
+          target/px-submission-tool-latest.zip
+          target/latest-version.txt
+        draft: false
+        prerelease: false
+        generate_release_notes: true
+        body: |
+          ## 🚀 PX Submission Tool ${{ github.ref_name }}
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)